spec/Vehicle/Vehicle.vspec

```diff
@@ -17,131 +17,134 @@
 
 VersionVSS.Minor:
   datatype: uint32
   type: attribute
   default: 0  
-  description: Supported Version of VSS - Minor version
+  description: Supported Version of VSS - Minor version.
 
 VersionVSS.Patch:
   datatype: uint32
   type: attribute
   default: 0
-  description: Supported Version of VSS - Patch version
+  description: Supported Version of VSS - Patch version.
 
 VersionVSS.Label:
   datatype: string
   type: attribute
   default: "develop"
-  description: Label to further describe the version
+  description: Label to further describe the version.
 
 #
 # Vehicle identification attributes.
 #
 
 VehicleIdentification:
   type: branch
-  description: Attributes that identify a vehicle
+  description: Attributes that identify a vehicle.
 
 VehicleIdentification.VIN:
   datatype: string
   type: attribute
-  description: 17-character Vehicle Identification Number (VIN) as defined by ISO 3779
+  description: 17-character Vehicle Identification Number (VIN) as defined by ISO 3779.
 
 VehicleIdentification.WMI:
   datatype: string
   type: attribute
-  description: 3-character World Manufacturer Identification (WMI) as defined by ISO 3780
+  description: 3-character World Manufacturer Identification (WMI) as defined by ISO 3780.
 
 VehicleIdentification.Brand:
   datatype: string
   type: attribute
-  description: Vehicle brand or manufacturer
+  description: Vehicle brand or manufacturer.
 
 VehicleIdentification.Model:
   datatype: string
   type: attribute
-  description: Vehicle model
+  description: Vehicle model.
 
 VehicleIdentification.Year:
   datatype: uint16
   type: attribute
-  description: Model year of the vehicle
+  description: Model year of the vehicle.
 
 
 #
-# Imports from schema.org/Car
-#
+# Imports from https://schema.org/Car
+# Names from https://schema.org/Car reused, but first character capitalized
+# Example: Original schema.org name: https://schema.org/acrissCode
+#          Name used in VSS: VehicleIdentification.AcrissCode
+# Note: Only a subset of the properties from https://schema.org/Car have been imported.
 
-VehicleIdentification.ACRISSCode:
+VehicleIdentification.AcrissCode:
   datatype: string
   type: attribute
   description: The ACRISS Car Classification Code is a code used by many car rental companies.
 
-VehicleIdentification.bodyType:
+VehicleIdentification.BodyType:
   datatype: string
   type: attribute
   description: Indicates the design and body style of the vehicle (e.g. station wagon, hatchback, etc.).
 
-VehicleIdentification.dateVehicleFirstRegistered:
+VehicleIdentification.DateVehicleFirstRegistered:
   datatype: string
   type: attribute
-  description: The date of the first registration of the vehicle with the respective public authorities.
+  description: The date in ISO 8601 format of the first registration of the vehicle with the respective public authorities.
 
-VehicleIdentification.meetsEmissionStandard:
+VehicleIdentification.MeetsEmissionStandard:
```

Comment:
This may need rework in another PR. Waht is meant with this? THe string should be EuroV/Euro5 oder something like that? Maybe needs to be clarified more. But won't fix in this PR

Reply:
Like all signals here it comes from schema.org (https://schema.org/docs/auto.home.html) and they do not specify it any further (https://schema.org/meetsEmissionStandard), seems to be a free text field that could contain text, URLs or anything.

But in general it seems like not much is happening in this area, I do not know if there are anyone actively maintaining the auto-related signals in schema.org. There is a W3C group but it seems to be quite dead. If no one use or maintain it we can consider removing the signals.

http://www.automotive-ontology.org/
https://www.w3.org/community/gao/

Reply:
Having an attribute describing emission standard might still make sense. I just fear it would be quite some work to collect all possible allowed values. P19 https://www.continental-automotive.com/getattachment/8f2dedad-b510-4672-a005-3156f77d1f85/EMISSIONBOOKLET_2019.pdf/  gives some hint towards just the more recent ones...